crew

```diff
@@ -531,14 +581,14 @@ when "whatprovides"
   if @pkgName
     whatprovides @pkgName
   else
-    puts "Usage: crew whatprovides [pattern]"
+    help @pkgName = "whatprovides"
```

Comment:
Once you use a local variable in help function, you can pass it here like `help "whatprovides"`, here and below.